### DISCUSSION TITLE
[SOLVED] Assign label trough Articles Filters

### DISCUSSION THREAD

**Discussion by BobSquarePants** [martinrotter/rssguard#770] [Q&A] [answered]
Hi everyone,

I'm reading the documentation @ https://github.com/martinrotter/rssguard/blob/master/resources/docs/Documentation.md#fltr

and I try to assign a label to articles when the match a word in title or body.

in the example we can see : `console.log()` great... but I didn't found a way to display the console !?

Cheers

**Comment by martinrotter** [comment, marked as answer]
Hi, `console.log()` prints messages into RSS Guard process command line console (standard output).

So to see "console" you must launch RSS Guard from command line (Linux). On Windows this approach is not possible right now as Windows does not allow (by default) to display console Window of a GUI application.

On windows launch RSS guard with parameter "--log", like this "rssguard.exe --log .\rssguard.txt". Then all logs will be written to "rssguard.txt". Open the file in some text editor which auto reloads file upon changes (like Notepad++ i guess) and then do your article filter debugging.

I might ease this in the future.

**Reply by BobSquarePants** [reply]
Thanks @martinrotter  !

**Comment by BobSquarePants** [comment]
Too bad when we hit **test** it will run it on all the **Existing articles**. 
![3](https://user-images.githubusercontent.com/23012674/182121582-7121c70e-1026-45e8-a98f-563705ee4946.png)
then the log quickly too long :/


It could be nice if we could select few articles to run the test
@martinrotter

**Reply by martinrotter** [reply]
You could click on some specific feed and only run on those articles.

I could maybe add checkboxes, to test only checked articles? If so, add relevant ticket.

**Reply by BobSquarePants** [reply]
It's up to you :) 
Actually, there is the **sample article** tab but the output seem not to mention everything (for example assigning a label)
